plugins/html/Dockerfile

```diff
@@ -1,25 +1,20 @@
-FROM rapid7/insightconnect-python-3-slim-plugin:5
+FROM --platform=linux/amd64 rapid7/insightconnect-python-3-slim-plugin:6.0.1
+
 LABEL organization=rapid7
 LABEL sdk=python
-LABEL type=plugin
-
-ENV SSL_CERT_FILE /etc/ssl/certs/ca-certificates.crt
-ENV SSL_CERT_DIR /etc/ssl/certs
-ENV REQUESTS_CA_BUNDLE  /etc/ssl/certs/ca-certificates.crt
 
-RUN apt-get update && apt-get install -y pandoc texlive lmodern --no-install-recommends
+WORKDIR /python/src
 
 ADD ./plugin.spec.yaml /plugin.spec.yaml
-ADD . /python/src
+ADD ./requirements.txt /python/src/requirements.txt
 
-WORKDIR /python/src
-# Add any package dependencies here
-
-# End package dependencies
 RUN if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+
+ADD . /python/src
+
 RUN python setup.py build && python setup.py install
 
-# needed for SSL
-USER root
```

Comment:
if this is needed for SSL then maybe we should keep this

Reply:
yep keep this as nobody, safe to change since theres no connection.
Not sure why this was root or the SSL comment was there to begin with